src/components/Map.vue

```diff
@@ -149,16 +155,29 @@ export default {
           buildTileUrlTemplate: this.buildTileUrlTemplate,
           crossOrigin: this.crossOriginMedia
         };
+
+        if (this.stac.type === 'Collection' && data.type === 'FeatureCollection') {
+          data = this.stac;
+          options.fillOpacity = 0;
+          this.collectionsLayer = await stacLayer(this.stacLayerData, {
+            fillOpacity: 0,
+            weight: 2,
+            color: '#188191',
+            displayPreview: this.stacLayerData.features.length < 50
```

Comment:
Maybe use the `itemsPerPage` config option as upper limit?

Reply:
Given that `itemsPerPage` can go up to 10,000 I suspect that's a bit high, so I've gone for an additional config option.

Reply:
Thanks. It should get a short explanation in thr Readme, too